crowdsec/config.yaml

```diff
@@ -13,6 +13,8 @@ arch:
   - aarch64
   - armv7
   - i386
+ports:
```

Comment:
You need to add the `ports_description` bellow `ports` section (following the [documentation](https://developers.home-assistant.io/docs/add-ons/configuration#optional-configuration-options)). So it will pass the add-on lint.

Reply:
Don't think this works. I manually changed everything 1.4.3 and it does not work. (Changing to 1.4.2 works fine)
The 1.4.3 will not start because it says config.plugin_dir is undefined. But I set it, then it gets wipes it out!

Reply:
I was talking only about the linting part. Indeed, I tested the changes in HAOS, and it was not working.
We figured out some issues in the source docker image that is used to build this one. So we fixed it, and we will do a new release asap.
I will add changes to this PR to have a final working image.